hips/hip-999.md

```diff
@@ -0,0 +1,105 @@
+---
+hip: 9999
+title: "Helm Org Maintainers and Deprecated Projects"
+authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+created: "2020-09-24"
+type: "process"
+status: "draft"
+---
+
+## Abstract
+
+This process proposal explains what happens to Helm organization maintainer slots when
+a project is deprecated.
+
+## Motivation
+
+The current governance documents state that org maintainers may be selected from any of
+the official Helm projects. However, as the Helm organization matures, some projects are
+being deprecated. The governance docs are unclear what happens to org maintainers from
+deprecated projects, and whether deprecated projects may hold org maintainer seats.
+
+## Rationale
+
+In the interest of keeping the Helm project thriving and alive, this HIP proposes that
+only non-deprecated projects are entitled to org maintainer seats.
+
+One project that is slated for deprecation is the charts repository, which previously had
+guaranteed org maintainers. This proposal updates that by splitting its designated spots
+between the remaining categories.
```

Comment:
The charts maintainers currently have 15 repositories. The _charts_ and _hub_ repos, owned by the charts maintainers are set to be archived. But, the 13 others are still in action.

There are two things to this:

1. The scope of the charts team and charts maintainers has shifted from the _charts_ repo to tooling that enables chart management.
2. There has been a proliferation of repos for tools to support charts and chart repos run by others. There is no documented process surrounding repos as opposed to projects which do not have a 1:1 mapping.

I would suggest the _charts_ repo is being deprecated and archived which the _charts team's_ purpose has shifted but it is not going away. I think both of these need to be codified in the governance in some form.

Reply:
I disagree with your assertion. The governance documentation singled out two repos: "helm/helm" and "helm/charts". It gave those two repos special consideration. It did not create a new category of maintainer called "chart maintainer" that therefore got multiple projects.

Reply:
The original commit (https://github.com/helm/community/commit/bbb28ea2fea54d44a391c36971a2e6c7d98a7143#diff-bcef1ca4b17b5dcae6fd40f5d117cb21) specifically says,

> Project maintainers focus on a single codebase or a group of related codebases. For example, the chart maintainers manage the charts repository and a few repositories that support charts.

and

> In some cases a groups of maintainers are responsible for more than one repo (e.g., charts maintainers managing the charts, chart-testing, charts-tooling). In other cases the maintainers are responsible for a single project (e.g., chartmuseum or monocular).

These are both in the original commit for the governance.